render/reader.go

```diff
@@ -0,0 +1,36 @@
+package render
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+)
+
+type Reader struct {
+	ContentType   string
+	ContentLength int64
+	Reader        io.Reader
+	Headers       map[string]string
+}
+
+// Render (Reader) writes data with custom ContentType and headers.
+func (r Reader) Render(w http.ResponseWriter) (err error) {
+	r.WriteContentType(w)
+	r.Headers["Content-Length"] = fmt.Sprintf("%d", r.ContentLength)
```

Comment:
Why not use `strconv.Itoa(r.ContentLength)`? Thanks!

Reply:
You're right. It's better to use `strconv` library but to convert a int64 variable to a string  `strconv.FormatInt(r.ContentLength, 10)` works better ;)

Reply:
Good!